### PR TITLE
refactor(sms-erros.js): Solves #5708 removed sms-errors.js

### PR DESCRIPTION
**Pull request by shikhar-scs**
Solves #5708 
As suggested by @shane-tomlinson [here](https://github.com/mozilla/fxa-content-server/issues/5708) all the sub-issues have been fixed which include

1. The entire sms-errors.js file was deleted.

2. The conversion [here](https://github.com/mozilla/fxa-content-server/blob/41add48551e4bd798c6cb9d9cae9a45f9a6f0790/app/scripts/lib/fxa-client.js#L791) was removed in fxa-client.js located at /app/scripts/lib/fxa-client.js

3. The checks/tests [here](https://github.com/mozilla/fxa-content-server/blob/41add48551e4bd798c6cb9d9cae9a45f9a6f0790/app/scripts/views/sms_send.js#L173) were removed from sms_send.js located at /app/scripts/views/sms_send.js

4. The extra tests [here](https://github.com/mozilla/fxa-content-server/blob/41add48551e4bd798c6cb9d9cae9a45f9a6f0790/app/tests/spec/lib/fxa-client.js#L1272) and [here](https://github.com/mozilla/fxa-content-server/blob/master/app/tests/spec/views/sms_send.js#L222:L238) were removed too.

5. As a final measure SmsErrors.toError('THROTTLED') in [this test](https://github.com/mozilla/fxa-content-server/blob/41add48551e4bd798c6cb9d9cae9a45f9a6f0790/app/tests/spec/views/sms_sent.js#L124) was changed to be AuthErrors.toError('THROTTLED').

At 4 places sms-errors had been required and thus, they were removed too (to successfully run grunt lint )and finally authErrors was added at the file mentioned in 5. All of this is reflected in the diff generated.

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 25b27b1c01c7764dca8aa9b99113535dde25a64a
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 9ba5ae60234dffdb9c83123d53c055afe861eace
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/mozilla/fxa-content-server/blob/master/CONTRIBUTING.md#git-commit-guidelines

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 25b27b1c01c7764dca8aa9b99113535dde25a64a
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 9ba5ae60234dffdb9c83123d53c055afe861eace
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 99eafeeebc26990134e2756f8f58de56b1e6d173
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/mozilla/fxa-content-server/blob/master/CONTRIBUTING.md#git-commit-guidelines

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 3c6322b33976128491debdef216c0cd881283dd5
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/mozilla/fxa-content-server/blob/master/CONTRIBUTING.md#git-commit-guidelines

---
This message was auto-generated by https://gitcop.com

**Comment by shikhar-scs** [comment]
@shane-tomlinson @vladikoff 
could you please suggest me something to do about the large amount of conflict errors in npm-shrinkwrap.json

File: package.json [hunk 1]
```diff
@@ -43,10 +43,10 @@
     "cors": "2.8.4",
     "express": "4.16.2",
     "extend": "3.0.1",
-    "fxa-shared": "1.0.4",
     "fxa-geodb": "0.0.8",
+    "fxa-shared": "1.0.4",
```
Comment (vladikoff) [inline package.json]: @shikhar-scs just reset the git changes made to `package.json` and the shrinkwrap file above. Your PR is not changing any dependencies

Comment (shikhar-scs) [reply package.json]: @vladikoff yeah sure will do

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 74273a05eb096737ae8f9cd438e669aa5d5f3dd3
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/mozilla/fxa-content-server/blob/master/CONTRIBUTING.md#git-commit-guidelines

---
This message was auto-generated by https://gitcop.com

**Comment by shikhar-scs** [comment]
@vladikoff I have committed again keeping in mind the changes suggested by you.
Also, there are no merge errors in npm-shrinkwrap.json file now

> just reset the git changes made to package.json and the shrinkwrap file above. Your PR is not changing any dependencies

This really helped. Thanx a lot .

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 74273a05eb096737ae8f9cd438e669aa5d5f3dd3
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 35c1a3f543ba5ef825dbd9d85d8a7ca88261520c
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/mozilla/fxa-content-server/blob/master/CONTRIBUTING.md#git-commit-guidelines

---
This message was auto-generated by https://gitcop.com

**Comment by shikhar-scs** [comment]
@vladikoff Initally when package.json had the following line included, the build on travis-ci was passing 
`"fxa-shared": "1.0.4"`
However, after removing that line the build fails every time . Should I re include that line in package.json and try again?

**Comment by shikhar-scs** [comment]
@vladikoff any updates?

**Comment by shikhar-scs** [comment]
@vladikoff mind reviewing here ?

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 74273a05eb096737ae8f9cd438e669aa5d5f3dd3
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 35c1a3f543ba5ef825dbd9d85d8a7ca88261520c
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: f7b1d97ed82ddea1c49760a4a3756d01f6a0530d
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/mozilla/fxa-content-server/blob/master/CONTRIBUTING.md#git-commit-guidelines

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 74273a05eb096737ae8f9cd438e669aa5d5f3dd3
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 35c1a3f543ba5ef825dbd9d85d8a7ca88261520c
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: f7b1d97ed82ddea1c49760a4a3756d01f6a0530d
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: bcc68b3f22939b21eba3908d9bfe15f16a259cd9
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/mozilla/fxa-content-server/blob/master/CONTRIBUTING.md#git-commit-guidelines

---
This message was auto-generated by https://gitcop.com

**Comment by shikhar-scs** [comment]
@vladikoff please help

**Comment by vladikoff** [comment]
@shikhar-scs sorry we were all away on Holidays, checking now...

**Pull request closed without merging**